frontend/src/components/contact/hooks/useAddContact.ts

```diff
@@ -0,0 +1,90 @@
+import axios, { AxiosError } from 'axios';
+import { useApiContacts } from 'hooks/pims-api/useApiContacts';
+import { ICreateOrganization, IEditablePerson } from 'interfaces/editable-contact';
+import { IApiError } from 'interfaces/IApiError';
+import { useDispatch } from 'react-redux';
+import { hideLoading, showLoading } from 'react-redux-loading-bar';
+import { toast } from 'react-toastify';
+import { logError } from 'store/slices/network/networkSlice';
+
+/**
+ * hook that adds a contact.
+ */
+const useAddContact = () => {
+  const { postPerson, postOrganization } = useApiContacts();
+  const dispatch = useDispatch();
+
+  const addPerson = async (
```

Comment:
Maybe I'm misunderstanding, but I don't think the ability to add a contact is part of this story? This story should be creating a component that allows a contact to be added to another entity, such as a deposit.

Reply:
that is true. I just moved all the hooks to the new location. I will remove the ones that are not used by the component directly and place them in their old place